src/components/TopBar/ColorBoard.tsx

```diff
@@ -1,14 +1,46 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { Box, Flex } from "@chakra-ui/react";
 import mockPlateColors from "./colorList";
-import { useDispatch } from "react-redux";
 import { changeSelectedColor } from "@/store/reducer/courtColorSlice";
+import { useDispatch } from "react-redux";
+import { useStoreSelector } from "@/store/hooks";
 
 const ColorBoard: React.FC = () => {
   const dispatch = useDispatch();
-  const handleChangeColor = (color: string) => {
+  const [fillColor, setFillColor] = useState<string>("#344c5c");
+  const handleChangeColor = (color: string): void => {
+    setFillColor(color);
     dispatch(changeSelectedColor(color));
   };
+  const { selectedColor } = useStoreSelector((state) => state.courtColor);
+
+  useEffect(() => {
```

Comment:
Could you please let me know why need to use useEffect?

Reply:
Coz it will have runtime error when SVG convert to CUR. So I found this method to fix it.